scripts/build/build.mk

```diff
@@ -34,9 +34,9 @@ deps:
 	$(GO_GET) -v -t -d ./...
 
 .PHONY: $(PLATFORMS)
-$(PLATFORMS): deps
+$(PLATFORMS): deps generate
```

Comment:
Please remove the `generate` command in the building process. Users have already followed the [compile documentation](https://github.com/apache/skywalking-rover/blob/main/docs/en/guides/compile/how-to-compile.md) to build the binary.

Reply:
Ok